src/components/notes/displayNote.js

```diff
@@ -1,41 +1,104 @@
 import React from "react";
 import { useNoteData, useArchive, useTrash } from "../../context/allContext";
-import { GetPriority, GetColor, getLabeledNotes} from "../../utils/filter-utils";
+import { GetPriority, GetColor, getLabeledNotes, DateSort } from "../../utils/filter-utils";
 
 export const DisplayNote = () => {
-  const { note } = useNoteData();
+  const { note, dispatch, setNoteEdit } = useNoteData();
   const { ArchiveNote } = useArchive();
   const { TrashNote } = useTrash();
-
   const compose = (...getNoteCard) => (note) => getNoteCard.reduce((data, getNoteCard) => getNoteCard(data), note);
-  const filteredProduct = compose(GetPriority, GetColor, getLabeledNotes)(note)
+  const filteredNotes = compose(GetPriority, GetColor, getLabeledNotes, DateSort)(note)
```

Comment:
I liked how you used compose here to add feature to note

Reply:
Thanks!